instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy/patches/client.rb

```diff
@@ -131,6 +132,10 @@ def database_name
             connection_options[:database]
           end
 
+          def database_user
+            connection_options[:username]
```

Comment:
I think this is here https://github.com/trilogy-libraries/trilogy/blob/d7da1848c50bca69bccaadc3eed1b51bf8e1da72/contrib/ruby/ext/trilogy-ruby/cext.c#L444